toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -524,7 +524,7 @@ func customizeOSContents(ctx context.Context, rc *ResolvedConfig) (imageMetadata
 		}
 	}
 
-	if len(rc.Config.Storage.Verity) > 0 || len(im.baseImageVerityMetadata) > 0 {
+	if len(rc.Storage.Verity) > 0 || len(im.baseImageVerityMetadata) > 0 {
```

Comment:
There are still a lot of references to `Config.Storage` in the codebase. These need to be redirected to `ResolvedConfig.Storage` instead. In VS Code, you can right click a field and select `Find All References`.

Also, you'll probably want to delete the `Config.CustomizePartitions()` function, since that contains a hidden reference to `Config.Storage`.

Also, the fact that you missed so much means that you need to expand your test coverage.